.github/workflows/docker-openmetadata-ingestion.yml

```diff
@@ -50,5 +54,5 @@ jobs:
           platforms: linux/amd64,linux/arm64
           push: ${{ github.event_name == 'release' }}
           # Update tags before every release
-          tags: 'openmetadata/ingestion:0.10.0,openmetadata/ingestion:latest'
+          tags: 'openmetadata/ingestion:${{ github.event.inputs.tag }},openmetadata/ingestion:latest'
```

Comment:
`github.event.inputs.tag` will be null in case of Github CI triggered by Release published event!